docs/third-party-developers/extensibility/data-store/schema.md

```diff
@@ -81,6 +84,47 @@ This will return all the registered routes for the given namespace as a flat arr
 
 -   `array`: An array of all routes for the given namespace.
 
+### getRouteFromResourceEntries
+
+This will returns the route from the given slice of the route state.
+
+#### _Parameters_ <!-- omit in toc -->
+
+-   _stateSlice_ `object`: Slice of the route state from a given namespace and resource name.
+-   _ids_ `array`: An array of id references that are to be replaced in route placeholders. If the route does not have placeholders, this can be an empty array.
+
+#### _Returns_ <!-- omit in toc -->
+
+-   `string` | `null`: The route for the given resource entries, or null if no route is found.
```

Comment:
```suggestion
-   `string`: The route for the given resource entries, or an empty string if no route is found.
```

See the original function description:
```ts
/**
 * Returns the route from the given slice of the route state.
 *
 * @param {Object} stateSlice This will be a slice of the route state from a
 *                            given namespace and resource name.
 * @param {Array}  [ids=[]]   Any id references that are to be replaced in
 *                            route placeholders.
 *
 * @return {string}  The route or an empty string if nothing found.
 */
const getRouteFromResourceEntries = ( stateSlice, ids = [] ) => {
	// convert to array for easier discovery
	stateSlice = Object.entries( stateSlice );
	const match = stateSlice.find( ( [ , idNames ] ) => {
		return ids.length === idNames.length;
	} );
	const [ matchingRoute, routePlaceholders ] = match || [];
	// if we have a matching route, let's return it.
	if ( matchingRoute ) {
		return ids.length === 0
			? matchingRoute
			: assembleRouteWithPlaceholders(
					matchingRoute,
					routePlaceholders,
					ids
			  );
	}
	return '';
};
```